build/docker/intel-fpga-initcontainer.Dockerfile

```diff
@@ -64,4 +64,4 @@ ln -sf $HOOK_CONF_DST /etc/containers/oci/hooks.d/$HOOK_CONF\n\
 rm $DST_DIR/deploy.sh\n\
 ">> $SRC_DIR/deploy.sh && chmod +x $SRC_DIR/deploy.sh
 
-CMD [ "/opt/intel/fpga-sw.src/deploy.sh" ]
+ENTRYPOINT [ "/usr/local/fpga-sw/deploy.sh" ]
```

Comment:
`fpga_admissionwebhook` suffers from the same issue

Reply:
@mythi thanks for pointing this out. Should be fixed now.